pkg/bff/organization.go

```diff
@@ -169,29 +171,64 @@ func (s *Server) ListOrganizations(c *gin.Context) {
 		return
 	}
 
+	// Parse the params from the GET request
+	params := &api.ListOrganizationsParams{}
+	if err = c.ShouldBindQuery(params); err != nil {
+		log.Warn().Err(err).Msg("could not bind request with query params")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(err))
+		return
+	}
+
+	// Set pagination defaults
+	if params.Page <= 0 {
```

Comment:
should this return an error instead of switching to the default?

Reply:
We could, in my opinion it increases the complexity of the design without providing much benefit. Recently I've been trying to follow the principle of minimizing the number of errors that we return to the client. In this case we can assume a default value so it's my preference to not generate an error out of nothing but I can also see the case where generating a 400 is perfectly valid.